docs/content/docs/getting_started/proxy_service_quickstart.adoc

```diff
@@ -60,11 +60,11 @@ providers:
 <2> Configures the `anonymous` authenticator.
 <3> Configures the `deny` authorizer, which rejects all requests.
 <4> Configures the `allow` authorizer. It will be used in our rule to allow requests.
-<5> Configures the `jwt` finalizer.
-<6> Configures the default rule.
-<7> Configures the `file_system` provider, which will allow loading of our rule from the file system.
+<5> Configures the `jwt` finalizer to create JSON Web Tokens.
+<6> Configures the default rule, which is applied if no other rule matches.
+<7> Configures the `file_system` provider, which will load rules from the file system.
 +
-Put together, this configuration will let heimdall create a JSON Web Token (JWT) with `sub` claim set to `anonymous` for every request on every URL for the HTTP methods GET and POST. The JWT itself will be put into the `Authorization` header as a bearer token. Since the above default rule cannot be used to forward requests to the upstream service, the `deny` authenticator will reject the requests for us and enforce a configuration of an authorization mechanism in a specific rule, which we're going to define next.
+Put together, this configuration will let heimdall create a JSON Web Token (JWT) with `sub` claim set to `anonymous` for every request on every URL for the HTTP methods GET and POST. The JWT itself will be put into the `Authorization` header as a bearer token. The above default rule cannot does not forward requests to the upstream service, instead it just rejects them with the `deny` authenticator. To forward requests to the upstream service, we need a configuration of an authorization mechanism in a specific rule, which we're going to define next.
```

Comment:
The sentence "The above default rule cannot does not forward requests ..." needs a small change ;)